### ISSUE TITLE
 Vulnerability to template injection that can lead to XSS.

### ISSUE THREAD

**Issue by danielaloycedaniel** [pillarjs/send#238]
It seems like this module contains vulnerabilities that may result in XSS attack according to npm audit report
```

# npm audit report

send  <0.19.0
Severity: moderate
send vulnerable to template injection that can lead to XSS - https://github.com/advisories/GHSA-m6fv-jmcg-4jfg
fix available via `npm audit fix`
node_modules/send
  serve-static  <=1.16.0
  Depends on vulnerable versions of send
  node_modules/serve-static

2 moderate severity vulnerabilities

To address all issues, run:
  npm audit fix
```

is there a solution?

**Comment by NewEraCracker** [comment]
See: https://github.com/pillarjs/send/issues/239#issuecomment-2399134135

I think this was solved with express 4.21.0

**Comment by UlisesGascon** [comment]
Yes @NewEraCracker! This was solved in 4.21.0 https://github.com/expressjs/express/releases/tag/4.21.0
